ksh93/PKGBUILD

```
@@ -0,0 +1,99 @@
+# This PKGBUILD is derived from one found on the AUR and includes patches contributed by gisburn.
+pkgname=ksh93
+pkgver=1.0.10
+pkgrel=1
+pkgdesc="The Original ATT Korn Shell, Now Maintained By The Community."
+arch=('i686' 'x86_64')
+url="https://www.github.com/ksh93/ksh"
+license=('spdx:EPL-2.0')
+makedepends=(
+	'gcc'
+  'gettext'	
+  'gettext-devel'
+	'ncurses-devel'
+	'gmp-devel'
+	'mpfr-devel'
+	'mpc-devel'
+	'isl-devel'
+)
```

there are more whitespace issues now.  For one thing, PKGBUILD files should indent with 2 spaces.  The arrays when split onto multiple lines should be lined up using spaces, like
```suggestion
makedepends=('gcc'
             'gettext'	
             'gettext-devel'
             'ncurses-devel'
             'gmp-devel'
             'mpfr-devel'
             'mpc-devel'
             'isl-devel')
```

also, your `ksh-mamake.patch` changed whitespace/indentation

---

I was puzzled why this was happening, but I discovered that because I was using nano, it by default inserted a literal tab which bonked with git when it was expecting spaces. I'll fix that.